CMSgov/mint-app
# [EASI-3713] - Frequency Updates FE Part 2

# EASI-3713

## Changes and Description

This PR is part 2 of all the frequency updates to forms.  The subsequent PRs will work of the groundwork laid here.
The first implementation is complete for  anticipatedPaymentFrequency question - [Figma](https://www.figma.com/file/M72DReE73SSXMN48WZfaKN/MINT---New-Model-Process-(Model-Intake)?type=design&node-id=7324-109204&mode=design&t=7zgFZTsLWqYWejIX-0)

- Added implementation of `FrequencyForm` to `participantsAndProviders.providerAdditionFrequency`
- Added reusable translation component for translating all `FrequencyTypeNew` enums
- Updated tests



## How to test this change

- Navigate to page 5 of 5 for participantsAndProviders (where providerAdditionFrequency question is rendered)
- Verify correct render and form functionality according to Figma
- Verify the data persists and renders correctly in the readonly view
- Verify the data renders in the CSV
- 
## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
